### ISSUE TITLE
detect result have no bbox

### ISSUE THREAD

**Issue by creabob** [amdegroot/ssd.pytorch#389]
when i use my own model to detect picture but it show nothing without bbox 
firstly, i think the object maybe too small 
secondly, when i train the model using default parameter show a nan loss so i change to 1e-7 and 16 batch size also i have try 1e-6 and 1e-8 the loss will between 150 to 250 and not converge . it is too high and i have no method to decrease the loss 
i will happy if u have some idea for me 
this is my picture.  my mission is  to detect rubbish on street
![298](https://user-images.githubusercontent.com/25314080/61994243-dbcbb480-b0aa-11e9-9bb2-227add518485.jpg)

**Comment by liujiale1234** [comment]
how to detect my own image??
I cant change the demo.py

**Comment by zhaohao0404** [comment]
> how to detect my own image??
> I cant change the demo.py

i also encounter the same conditions. hi, friend, have you solved it ?

**Comment by leiyaohui** [comment]
hello ,i also encounter the same conditions.have you solved it ?

**Comment by zhaohao0404** [comment]
> hello ,i also encounter the same conditions.have you solved it ?

if the finally mAP is affected by the batch_size with the same configuration?
